package.json

```diff
@@ -1,31 +1,26 @@
 {
-  "name": "@prosperworks/eslint-config-prosperworks",
-  "version": "1.1.2",
+  "name": "@prosperworks/eslint-config",
+  "version": "2.0.0",
   "description": "A mostly reasonable approach to JavaScript.",
-  "scripts": {
-    "prelint": "editorconfig-tools check ./rules/*",
-    "lint": "eslint ./rules",
-    "fix": "eslint ./rules --fix"
-  },
-  "repository": "git+https://git@github.com/ProsperWorks/js-style.git",
-  "homepage": "https://github.com/ProsperWorks/js-style",
-  "author": "Eric Peterson <eric@prosperworks.com>",
+  "repository": "github:ProsperWorks/js-style",
+  "author": "ProsperWorks",
   "license": "MIT",
   "keywords": [
-    "style guide",
-    "lint",
     "eslint",
-    "prosperworks",
-    "es6",
-    "es2017"
+    "eslintconfig",
+    "prosperworks"
   ],
   "devDependencies": {
-    "editorconfig-tools": "^0.1.1",
-    "eslint": "^3.19.0",
-    "eslint-find-rules": "^1.14.3",
-    "eslint-plugin-import": "^2.2.0"
+    "eslint": "^4.19.0",
+    "eslint-plugin-ember": "^5.1.0",
+    "eslint-plugin-import": "^2.11.0"
+  },
+  "peerDependencies": {
+    "eslint": "^4.19.0",
+    "eslint-plugin-ember": "^5.1.0",
+    "eslint-plugin-import": "^2.11.0"
```

Comment:
When eslint-configs contain rules for plugins, the documentation states they must list those plugins as peer dependencies, to alert projects that they should also install those dependencies. This is stupid and lots of ppl agree, but npm maintainers haven't budged in 3 years of debate on this topic so we'll just do it their stupid way.